core/unit_test/CMakeLists.txt

```diff
@@ -8,23 +8,23 @@ SET(GTEST_SOURCE_DIR ${${PARENT_PACKAGE_NAME}_SOURCE_DIR}/tpls/gtest)
 #need here for tribits
 KOKKOS_INCLUDE_DIRECTORIES(${GTEST_SOURCE_DIR})
 KOKKOS_ADD_TEST_LIBRARY(
-  kokkos_gtest
-  HEADERS ${GTEST_SOURCE_DIR}/gtest/gtest.h
-  SOURCES ${GTEST_SOURCE_DIR}/gtest/gtest-all.cc
```

Comment:
There are still all of these changes.

Reply:
When it's the last thing, I'm going to check out the version from develop, and just change on top of that

Reply:
Fixed via 254302a